pkgs/development/python-modules/adjustText/default.nix

```
@@ -0,0 +1,29 @@
+{ buildPythonPackage
+, numpy
+, matplotlib
+, fetchFromGitHub
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "adjustText";
```

```suggestion
  pname = "adjusttext";
```

Should follow PEP-503, see https://nixos.org/manual/nixpkgs/stable/#contributing

---

Unfortunately this is how the package is named on pypi. I agree it sucks but at the same time I think we should maintain consistency with pypi naming so that users will have the same experience as with pip install.

---

That is actually not true. New python packages in nixpkgs need to follow pep503 and the attr, pname and directory name must all be pep503 conform. The project name on pypi gets usually all lower cased and all underscores and dots are replaced with a minus.

---

Oh sorry, I wasn't aware of this policy. I assumed the link above was to the Python pip not the packaging best practices, and didn't investigate further. I'll update it to be the correct name.

---

Done in https://github.com/NixOS/nixpkgs/pull/137462/commits/975d350d51db0bcd956e5f57afd14ba53606144c.